docs/getting-started.md

```
@@ -0,0 +1,84 @@
+- Unity **2022.3** 以降（VRChat 推奨環境。例: 2022.3.22f1）
+- VRChat Creator Companion (VCC) または Git URL 経由でのインストール
+
+---
+
+## インストール
+
+### A. VCC (VRChat Creator Companion) 経由（推奨）
+
+1. VCC を開き、**Settings → Packages → Add Repository** を選択
+2. 以下の URL を追加:
+   ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)